src/BenchmarkDotNet.Core/Exporters/Xml/SimpleXmlWriter.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+
+namespace BenchmarkDotNet.Exporters.Xml
+{
+    public class SimpleXmlWriter : IXmlWriter, IDisposable
```

Comment:
I went through all the `XmlExporter`'s classes and I think I got them all changed to `internal` apart from the actual Exporter.